### PR TITLE
supervisor edit subject_task in each course

### PR DESCRIPTION
**Pull request by VuongLong**

### CODE REVIEW

File: app/helpers/rails_admin/application_helper.rb [hunk 1]
```diff
@@ -177,14 +177,15 @@ def flash_alert_class flash_key
       end
     end
 
-    def course_actions
+     def course_actions
```
Comment (trantube94) [inline app/helpers/rails_admin/application_helper.rb]: indent

**Comment by lannp** [comment]
OK

File: app/decorators/task_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class TaskDecorator < Draper::Decorator
+  delegate_all
+
+  def get_name
+    name.nil? ? TaskMaster.find(task_master_id).name : name
```
Comment (dieunb) [inline app/decorators/task_decorator.rb]: name || TaskMaster.find(task_master_id).name

File: app/decorators/task_decorator.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  end  
+
+  def get_description
+    description.nil? ? TaskMaster.find(task_master_id).description : description
```
Comment (dieunb) [inline app/decorators/task_decorator.rb]: `description || TaskMaster.find(task_master_id).description`

**Pull request merged**